app/src/main/java/com/realityexpander/tasky/presentation/login_screen/LoginScreen.kt

```diff
@@ -57,113 +86,201 @@ fun LoginScreen(
     }
 
     BackHandler(true) {
-        navigateToRegister()
+        /* do nothing, should we ask the user to quit? */
```

Comment:
Why do you disallow to close the app here?

Reply:
I notice that most apps dont let you quit, and I was trying to follow that standard. But I will allow the user to quit by pressing the back button.